indy-repository-manager/src/test/java/org/jboss/pnc/indyrepositorymanager/AbstractRepositoryManagerDriverTest.java

```diff
@@ -53,6 +53,7 @@
     protected static final String CONFIG_SYSPROP = "pnc-config-file";
 
     protected static final String PNC_BUILDS = "pnc-builds";
+    protected static final String BREW_PULL_ACTIVE_METADATA_KEY = "koji-active";
```

Comment:
According to the latest comments at MMENG-1262 the key will be "koji-pull". Not sure if it matters to the tests though.

Reply:
Thanks for letting me know! I updated the field to minimize confusion to future readers of this code.